frontend/src/main/java/puf/frisbee/frontend/model/PlayerModel.java

```diff
@@ -59,15 +64,45 @@ public boolean register(String name, String email, String password){
                     .build()
                     .send(request, HttpResponse.BodyHandlers.ofString());
 
-            return response.statusCode() == 201 ? true : false;
+            return response.statusCode() == 201;
         }catch (Exception e){
             return false;
         }
     }
 
     @Override
     public boolean login(String email, String password) {
-        return email.equals(this.email) && password.equals(this.password);
+
+        try {
+            String loginCredentials = "{\"email\":\"" + email + "\",\"password\":\"" + password  +"\"}";
```

Comment:
Geht das mit Java so, mit den JSONs 🤔 ? Ich will es nicht komplizierter machen, aber das sieht hacky aus :D ... aber vllt ist das so auch ok.

Reply:
Das geht mit Java leider gar nicht in-built :disappointed: Für komplette Objekte lässt sich der Jackson ObjectMapper nutzen aber bei einzelnen Strings wirds schwierig. Ich hab da nichts besseres gefunden. In vielen Beispielen wurde das auch einfach so in einen String geschrieben :thinking:

Reply:
Oke, dann lassen wir es erstmal so... ich hatte noch https://www.baeldung.com/java-org-json gefunden, aber wahrscheinlich ist das einfach nur unnötiger overhead. Sollten wir mal komplexere JSONs brauchen, dann können wir das ja immer noch nehmen.

Reply:
Komplexe JSONs kann Jackson auch, denke ich: https://www.baeldung.com/jackson-object-mapper-tutorial Da könnten wir uns dann auch überlegen, einen custom serializer/deserializer zu schreiben, falls benötigt.